site/content/faq/what-is-the-default-directory-that-zap-uses.md

```diff
@@ -25,6 +25,6 @@ Note: ZAP's home is not the same as the user's home directory, which is also OS
 
 `~/Library/Application Support/ZAP`
 
-##  Weekly releases
+##  Weekly releases and development builds
 
-Note that for weekly releases `_D` is appended to the above directory. Development builds also use this directory.
+Note that for weekly releases `_D` is appended to the above directory. 
```

Comment:
IMO this should be tweaked too, to keep it clear that both weekly and development builds use the directory.

Reply:
Ya it could just be a unified statement, like:
> Note that for weekly releases and dev builds `_D` is appended to the above mentioned directory name.

Reply:
```suggestion
Note that `_D` is appended to the above directory. E.g. in Linux the zap.log file will be saved in ~/.ZAP_D
```

Reply:
```suggestion
Note that `_D` is appended to the above directory. E.g. in Linux the home directory is ~/.ZAP_D
```